depends/packages/libmultiprocess.mk

```diff
@@ -3,7 +3,7 @@ $(package)_version=$(native_$(package)_version)
 $(package)_download_path=$(native_$(package)_download_path)
 $(package)_file_name=$(native_$(package)_file_name)
 $(package)_sha256_hash=$(native_$(package)_sha256_hash)
-$(package)_dependencies=native_$(package) boost capnp
+$(package)_dependencies=native_$(package) capnp
```

Comment:
I tried to build locally `multiprocess` from branch - it succeed.

Btw, maybe we should enable back multiprocess?
Does it worth burned CPU time?
if so, then https://github.com/knst/dash/commit/eb044d947b45099b3244daedc26a32e006ad0c6c

Reply:
sooo should I drop or nah?

Reply:
it succeed on my localhost (dunno about CI), but we have no actually multiprocess yet. I guess no need to drop if it works, because it is easy to fix anyway later: the failure is clear "build failed because no cap" if anything indeed broken.

I vote for not removing now; and not enabling CI before some more progress for multiprocess is done.

Reply:
Okay; Please utACK then :D @UdjinM6 please disagree if you'd like :)